janusgraph-test/src/test/java/org/janusgraph/graphdb/serializer/attributes/THashMapSerializer.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.graphdb.serializer.attributes;
+
+import java.util.HashMap;
+
+import org.janusgraph.diskstorage.ScanBuffer;
+import org.janusgraph.graphdb.database.serialize.attribute.SerializableSerializer;
+
+/**
+ * @author Scott McQuillan (scott.mcquillan@uk.ibm.com)
+ */
+public class THashMapSerializer extends SerializableSerializer<HashMap<String, ?>> {
```

Comment:
Drop `<String, ?>` (this allows `serialize.registerClass(1001, HashMap.class, new THashMapSerializer())`).